helpdesk/settings.py

```diff
@@ -56,6 +56,11 @@
                                          'HELPDESK_STAFF_VIEW_PROTECTOR',
                                          lambda _: None)
 
+# Enable ticket and Email attachments
+HELPDESK_ENABLE_ATTACHMENTS = getattr(settings,
+                                      'HELPDESK_ENABLE_ATTACHMENTS',
+                                      False)
+
```

Comment:
Unless there is a strong argument to changing a default, I will always err on the side of remaining backwards compatible.
In this case I can see the argument that there is a security risk for new users who do not want attachments enabled but that is still an installation decision new users should make and as such, as long as we clearly document it in the install instructions, I do not see that it is necessary to make the default "False". 
This means that long time existing users will not be surprised by a change they did not expect when they upgrade and find they have lost a chunk of data for their tickets before they realise what the problem is.
I am open to discussion on this if there are strong opinions for making it default to False.

Reply:
No strong argument at all.

I'm always happy to find, that makers of open source package 'X' have my back covered when I start dabbling with 'X', but I also understand, that breaking software with an update will always cause confusion and anger.

I propose to
- Expand the security topic in the documentation
- Set safety default in a future major upgrade with due announcements.

For now I will update the PR to default to enabled attachments.